src/config/dark-sites.config

```diff
@@ -649,6 +649,7 @@ jsongoku.com
 justfive.news
 justwatch.com
 kadantiscam.netlify.app
+kag.info.hu
```

Comment:
So a site doesn't qualify to the `dark-sites.config` if it has the `prefers-color-scheme` feature?

Reply:
It would take me too much time to find it but summarizing. We add URL to dark-sites.config only if it is "always" dark. If site has option to enable/disable dark mode then it's disqualified from adding here. Why?

Because some of users can use some systems that not have theming so the page can load as white and with added URL to dark sites Dark Reader will tell it's on global dark sites list.

Same if site doesn't have switches but it checks "prefers-color-scheme" from the system/browser. If anybody uses light theming but want to have dark pages (I know not many people would do that but they can 😝) then they end with white site and Dark Reader will tell it's on global dark sites list.

So summarizing... before adding anything here you need to check if it won't open white when system/browser is set to light mode + you need to be sure if it doesn't have mode switches like this I found.

P.S.
Still waiting for the screens of unipolls to confirm what is fixed there.

Reply:
> It would take me too much time to find it but summarizing. We add URL to dark-sites.config only if it is "always" dark. If site has option to enable/disable dark mode then it's disqualified from adding here. Why?
> 
> Because some of users can use some systems that not have theming so the page can load as white and with added URL to dark sites Dark Reader will tell it's on global dark sites list.
> 
> Same if site doesn't have switches but it checks "prefers-color-scheme" from the system/browser. If anybody uses light theming but want to have dark pages (I know not many people would do that but they can 😝) then they end with white site and Dark Reader will tell it's on global dark sites list.
> 
> So summarizing... before adding anything here you need to check if it won't open white when system/browser is set to light mode + you need to be sure if it doesn't have mode switches like this I found.
> 
> P.S. Still waiting for the screens of unipolls to confirm what is fixed there.

Thanks for the explanation, found the related part in the documentation that I haven't found it before so I deleted my initial question. 😅